packages/data-models/constants.ts

```diff
@@ -48,6 +48,11 @@ const APP_LANGUAGES = {
   default: "gb_en",
 };
 
+const APP_SKINS_CONFIG = {
```

Comment:
I'm not sure exactly what the `default` and `enabled` options refer to - can you add a couple comments to clarify?